src/modules/launcher/Wox/ViewModel/ResultViewModel.cs

```diff
@@ -22,7 +22,25 @@ public enum ActivationType
             Hover
         };
 
-        public List<ContextMenuItemViewModel> ContextMenuItems { get; set; }
+        private List<ContextMenuItemViewModel> _items = new List<ContextMenuItemViewModel>();
+
+        public List<ContextMenuItemViewModel> ContextMenuItems

```

Comment:
Is this list static? If not we can use ObservableCollection instead

Reply:
It is not a static list. I changed it to use ObservableCollection instead as you suggested. That way I don't have to call onPropertyChanged in the setter as it implements INotifyPropertyChanged. Thank you!